lib/client-assets.php

```diff
@@ -462,12 +462,12 @@ function gutenberg_extend_wp_api_backbone_client() {
 				return model.prototype.route && route === model.prototype.route.index;
 			} );
 		};
-		wp.api.getPostTypeRevisionsCollection = function( postType ) {
+		/*wp.api.getPostTypeRevisionsCollection = function( postType ) {
```

Comment:
We no longer need this handler, should we remove it?

Reply:
If it's dead code, I'm in favor of removing it.